cmd/toldata-gen/templates.go

```diff
@@ -38,17 +39,30 @@ type {{ $ServiceName }}REST struct {
 }
 
 func New{{ $ServiceName }}REST(ctx context.Context, config toldata.ServiceConfiguration) (*{{ $ServiceName }}REST, error) {
+	var service {{ $ServiceName}}REST
```

Comment:
As mentioned, REST and GRPC templates totally must not be touched.
Only the toldata API calls which are rerouted to the direct struct.